server/app/services/sentiment_analyzer.py

```
@@ -0,0 +1,331 @@
+            import nltk
+            
+            # Check if VADER lexicon exists (hard fail if not provisioned)
+            try:
+                nltk.data.find('sentiment/vader_lexicon.zip')
+            except LookupError:
+                error_msg = (
+                    "NLTK VADER lexicon not found. Please provision it before deployment:\n"
+                    "  python -m nltk.downloader vader_lexicon\n"
+                    "This is a required dependency and must be installed during container build."
+                )
+                logger.error(f"❌ {error_msg}")
```

Ruff (0.15.2)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)